### ISSUE TITLE
ToolTip "black tooltip" phenomenon

### ISSUE THREAD

**Issue by Nirus2000** [portfolio-performance/portfolio#4328]
Hello, @ma4nn @Nirus2000  do you have a "black tooltip" phenomenon with or without this PR ? 
On Windows 11 I have it sometimes. It was already present in some cases and in some chart, but wonder if with this change is impacting it even further. 

- Kommer master (left) vs Kommer 0.71.2 (right):

https://github.com/user-attachments/assets/9f7ccd4c-08da-4e20-8816-d116c907a835

But even more strange:

- Kommer 0.71.2, **so without this PR**: I have the blacktooltip with a specific data series:

https://github.com/user-attachments/assets/32404658-60c9-476c-bf7f-c23c242fceaa

Does this happen also for Mac ?

_Originally posted by @mierin12 in https://github.com/portfolio-performance/portfolio/issues/4297#issuecomment-2453120257_

**Comment by Nirus2000** [comment]
Hello @mierin12 
I took a look at the whole thing.
In version 0.71.2 I have the “black window” problem.
With the current master... as version 0.71.3.qualifier, the problem no longer exists.
_Microsoft Windows 10 Pro 22H2 (19045.5073)_

https://github.com/user-attachments/assets/3f7255a3-b220-4931-b187-ea893a85c8df

**Comment by mierin12** [comment]
So strange ! Maybe it depends somehow on the undelaying dataseries plotted.

**Comment by ma4nn** [comment]
Yeah, strange &#x1F914; It could be an a issue with the resizing, but that would not explain why it also occurs without [my change](https://github.com/portfolio-performance/portfolio/pull/4297). Unfortunately I cannot reproduce it on my Mac..
